var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -273,6 +273,9 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     patch("vtk-xdmf2-hdf51.13.1.patch", when="@5.10.0:5.10")
     patch("vtk-xdmf2-hdf51.13.2.patch", when="@5.10:")
 
+    # Fix VTK to work with external freetype using CONFIG mode for find_package
+    patch("FindFreeType.cmake.patch", when="@5.10.1:")
```

Comment:
typo here, `Type` -> `type`

Reply:
Tried this on windows, so that did not matter. :-) Thanks!